control.lua

```diff
@@ -564,7 +565,10 @@ script.on_event(defines.events.on_force_created, on_force_created)
 
 script.on_event(defines.events.on_tick, on_tick)
 script.on_event(defines.events.on_gui_click, on_gui_click)
-script.on_event(defines.events.on_gui_checked_state_changed, on_gui_click)
+
+-- TODO: remove
+-- why do we need this?
+-- script.on_event(defines.events.on_gui_checked_state_changed, on_gui_click)

```

Comment:
That was added somewhere shortly after the 0.16 release, since clicking a checkbox wouldn't trigger the on_gui_click event anymore. Seems that is no longer the case